db/migrate/20200712150048_devise_invitable_add_to_users.rb

```diff
@@ -0,0 +1,32 @@
+class DeviseInvitableAddToUsers < ActiveRecord::Migration[6.0]
+  def up
+    safety_assured do
+      change_table :users do |t|
+        t.string     :invitation_token
+        t.boolean    :registered, default: true
```

Comment:
**question:** Adding default values on big tables can be fairly disruptive, have we considered this?

Reply:
The `users` table isn't _that_ big at ~425k rows or so. And from my understanding this is much more problematic with the modification of existing columns vs adding new ones. I _think_ we're good here.

Reply:
should we add an index on `registered` ? The code is doing a lot of `.where(registered: true|false)` queries

Reply:
@rhymes My understanding is that indexes on booleans wouldn't add a lot of value.

> PostgreSQL will use an index only if it thinks it will be cheaper that way. An index on a boolean column, which can only take two possible values, will almost never be used, because it is cheaper to sequentially read the whole table than to use random I/O on the index and the table if a high percantage of the table has to be retrieved.

[source](https://stackoverflow.com/questions/42972726/postgresql-create-index-for-boolean-column)

Reply:
TIL! :-)

Reply:
> And from my understanding this is much more problematic with the modification of existing columns vs adding new ones.

I agree with this statement, but I think it never hurts to bring it up in PR reviews.